frontend/src/utils/index.js

```diff
@@ -758,7 +758,7 @@ export function calculateRelatedShootCount (shootList, binding) {
     return shootsByInfrastructureBinding.length
   } else if (binding._isDnsBinding) {
     if (!binding._secretName) {
-      return 0 // currently DNS provider does not support workload identity
+      return 0 // currently DNS provider only allows to reference secrets directly
```

Comment:
use 
`// dns extension currently supports secrets only (no bindings)`
comment (consistent with comment in credentials store)